tests/unittest.cpp

```diff
@@ -979,6 +987,66 @@ TEST_CASE("GenericParameters return types", "[generic-parameters][static-checks]
                                                                                                          // strings
 }
 
+TEST_CASE("Missing files (ROOT readers)", "[basics]") {
+  bool exc = false;
+  auto root_reader = podio::ROOTReader();
+  try {
+    root_reader.openFile("NonExistentFile.root");
+  } catch (const std::runtime_error& e) {
+    exc = true;
+  }
+  REQUIRE(exc);
```

Comment:
```suggestion
  auto root_reader = podio::ROOTReader();
  REQUIRE_THROWS(root_reader.openFile("NonExistantFile.root"));
```
This should work (if I read the [documentation](https://github.com/catchorg/Catch2/blob/devel/docs/assertions.md#exceptions) correctly) and is much cleaner, IMHO. In principle one could even make sure that the exception message indeed contains something pointing to a non-existant file, but I don't think that is necessary.